pkgs/development/compilers/ghcjs/base.nix

```diff
@@ -159,6 +163,7 @@ in mkDerivation (rec {
         --with-cabal ${cabal-install}/bin/cabal \
         --with-gmp-includes ${gmp.dev}/include \
         --with-gmp-libraries ${gmp.out}/lib
+      ${if isNull ghcLibdir then "" else ''echo "${ghcLibdir}" > "$out/lib/ghcjs-${version}/ghc_libdir"''}
```

Comment:
The [other commit](https://github.com/NixOS/nixpkgs-channels/commit/1cd07402f168329d7104e72e6cd4158f19de0b80) doesn't provide any further details, I'm afraid (just says "wip").